imagescan/delta.go

```diff
@@ -218,11 +235,6 @@ func (d *deltaState) handlePodDelete(pod *corev1.Pod) {
 				delete(img.owners, ownerResourceID)
 			}
 		}
-
-		if len(img.owners) == 0 {
```

Comment:
After we remove this, is there any reason to still keep image owner references?

Reply:
we use owner references are used when deciding if scan should be performed